### PR TITLE
Release notes for 17-11-0

### PR DESCRIPTION
**Pull request by mosesn**

### CODE REVIEW

File: source/blog/2017-11-17-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    for exceptions that are commonly retried when building from ClientBuilder but had
+    no MethodBuilder equivalents. [3d292b49](https://github.com/twitter/finagle/commit/3d292b49e737538babb387ae39e6ee9124a641ed)
+
+  * finagle-netty4: `Netty4Transporter` and `Netty4Listener` are now accessible, which
+    allows external users to create their own protocols for use with Finagle on Netty 4.
+    [3ce475db](https://github.com/twitter/finagle/commit/3ce475dbdb800a635bbed944139b9690418e09d9)
+
+Bug Fixes:
+
+  * finagle-exp: Fix race condition in `LatencyHistogram` which could lead to the wrong
```
Comment (kevinoliver) [inline source/blog/2017-11-17-release-notes.md]: this is still broken though and we will be replacing it whole cloth soon. mind omitting from here?

File: source/blog/2017-11-17-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+title: 🍠 November 2017 Release Notes - Version 17.11.0
+published: true
+post_author:
+  display_name: Moses Nakamura
+  twitter: mnnakamura
+tags: Releases, Finagle, Finatra, Util, Scrooge, TwitterServer
+---
+
+It's November, and you know what that means!  Library releases!!!  We're now publishing
+with sbt 1.0.x, and we now also publish sbt-scrooge-plugin for sbt 1.0.x.
```
Comment (cacoco) [inline source/blog/2017-11-17-release-notes.md]: "..and the sbt-scrooge-plugin additionally for sbt 1.0.x" ?

just want to make it clear it's being publishing in addition for sbt 1.0.x as we still publish for sbt 0.13.x

File: source/blog/2017-11-17-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: 🍠 November 2017 Release Notes - Version 17.11.0
```
Comment (cacoco) [inline source/blog/2017-11-17-release-notes.md]: +1 emoji.

Comment (nepthar) [reply source/blog/2017-11-17-release-notes.md]: 👏

**Pull request merged**